defold-spine/src/comp_spine_model.cpp

```diff
@@ -306,9 +306,16 @@ namespace dmSpine
     }
 
     static bool PlayAnimation(SpineModelComponent* component, dmhash_t animation_id, dmGameObject::Playback playback,
-        float blend_duration, float offset, float playback_rate, int track_index)
+        float blend_duration, float offset, float playback_rate, int track_index, bool is_initial = false)
     {
         uint32_t index = FindAnimationIndex(component, animation_id);
+        
+        // Ignore error for empty animation on spine creation 
+        if (index == INVALID_ANIMATION_INDEX && is_initial && animation_id == dmHashString64(""))
```

Comment:
For constant hashes we usually put them as constants at global scope, as you see at the top of the file.

Also, the hash of "" is 0.